packages/ui/src/lib/media-query.test.ts

```diff
@@ -28,13 +28,23 @@ describe('media-query', () => {
       // align
       const breakpoint = 'smurf'
 
-      // Suppress log
-      jest.spyOn(global.console, 'error').mockImplementationOnce(() => {})
+      // prevent writing to stderr during this render
+      const err = console.error
+      console.error = jest.fn()
 
       // act
-      const { result } = renderHook(() => useBreakpoint(breakpoint as Level))
+      let exception: unknown
+      try {
```

Comment:
I guess one could also use


```suggestion
      expect(() =>  {
          renderHook(() => useBreakpoint(breakpoint as Level))
          }).toThrow(`Unknown breakpoint ${breakpoint}`)
```

https://jestjs.io/docs/expect#tothrowerror

Reply:
yes, I thought so but it fails when I try to call it that way :/